admin/src/js/controllers/upgrade.js

```diff
@@ -141,6 +144,7 @@ angular.module('controllers').controller('UpgradeCtrl',
         ])
         .then(([ branches, betas, releases, featureReleases ]) => {
           $scope.versions = { branches, betas, releases, featureReleases };
+          buildsDb.destroy();
```

Comment:
Actually... shouldn't I be using db.close() here? I wonder what the difference is for the http adapter.

Reply:
You're right, `destroy` tries to delete the database.
```
 api._destroy = function (options, callback) {
    fetchJSON(genDBUrl(host, ''), {method: 'DELETE'}).then(function (json) {
      callback(null, json);
    }).catch(function (err) {
      /* istanbul ignore if */
      if (err.status === 404) {
        callback(null, {ok: true});
      } else {
        callback(err);
      }
    });
  };
```

We got away with it because all our requests were unauthenticated :D